app/client/src/features/events/LiveFeedback/SubmitLiveFeedback.tsx

```diff
@@ -40,12 +43,19 @@ export function SubmitLiveFeedback({ className, eventId }: Props) {
     const [isOpen, open, close] = useResponsiveDialog();
     const [user] = useUser();
     const [commit] = useMutation<SubmitLiveFeedbackMutation>(SUBMIT_LIVE_FEEDBACK_MUTATION);
-
+    const { displaySnack } = useSnack();
+    
     function handleSubmit(form: TLiveFeedbackFormState) {
-        commit({
-            variables: { input: { ...form, eventId } },
-            onCompleted: close,
-        });
+        try {
+            if (form.message.length >= FEEDBACK_MAX_LENGTH) throw new Error('Question is too long!')
+            if (isURL(form.message)) throw new Error('No links are allowed!') 
```

Comment:
Hmm, would this catch if the URL is within a message?  For example, "Please take a look at [some url here]"

My understanding of the current implementation of `isUrl` is that it determines if the entire string is a URL or not.  Whereas the behavior we desire is checking whether or not any substring is a URL.

Reply:
right, but in that case we wouldn't be able to use the URL() function to check if it's a valid url. If the string is "hi google.com blah" then putting the whole string into URL() would throw an error. I suppose I could pass in substring word by word separated by whitespace into the URL() function (URL(hi), URL(google.com). However, this approach wouldn't work if the string was something like "higoogle.com". So, wouldn't regex be a more viable option to check for such cases

Reply:
isUrl is my function that I wrote with regex that I imported in from index.ts; it does in fact catch the example you mentioned, "Please take a look at [some url here]". I also verified some other cases as well ("higoogle.com"; without the whitespace also gets caught in the function)

Reply:
Okay, seems I was mistaken. Leave as-is.

Reply:
was there something else you wanted me to do for this issue?

Reply:
Nothing at the moment. I'll give the code a second pass in the next day or two.